src/models/notifications.js

```diff
@@ -32,7 +32,12 @@ function verifySignature (notification, config) {
 
 function * processNotification (notification, ledgers, config) {
   if (notification.event === 'transfer.update') {
-    yield payments.updateTransfer(
+    const ledger = ledgers.getLedger(notification.resource.ledger)
```

Comment:
That would be a breaking change. (This is a non-breaking PR.)

My guess is we'll probably just take the HTTP notifications out entirely. They are no longer used, but I kept them in because they are still used by the majority of our tests for the internal business logic. I added subscriptionSpec.js so that we have a new test suite that we can migrate those tests to (in future refactorings.)